data-model/digitalobjects/0.2.0/digital-specimens/schema/identifications.json

```diff
@@ -0,0 +1,365 @@
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment": "DigitalObject Version 0.2.0",
+  "type": "object",
+  "properties": {
+    "dwc:identificationID": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identificationID",
+      "examples": [
+        "9992"
+      ]
+    },
+    "???:identificationType": {
+      "type": "string",
+      "description": "Unclear yet",
+      "$comment": "Unknown what this field should be"
+    },
+    "???:taxonFormula": {
+      "type": "string",
+      "description": "Unclear yet",
+      "$comment": "Unknown what this field should be"
+    },
+    "dwc:verbatimIdentification": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/verbatimIdentification",
+      "examples": [
+        "Peromyscus sp."
+      ]
+    },
+    "dwc:typeStatus": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/typeStatus",
+      "example": [
+        "holotype"
+      ],
+      "$comment": "Unclear if we can make this an enumeration, which values?"
+    },
+    "dwc:identifiedBy": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identifiedBy",
+      "example": [
+        "James L. Patton"
+      ]
+    },
+    "dwc:identifiedById": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identifiedByID",
+      "example": [
+        "https://orcid.org/0000-0002-1825-0097"
+      ]
+    },
+    "dwc:dateIdentified": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/dateIdentified",
+      "example": [
+        "2009-02-20T08:40Z"
+      ]
+    },
+    "dwc:identificationReferences": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identificationReferences",
+      "example": [
+        "Aves del Noroeste Patagonico. Christie et al. 2004."
+      ],
+      "$comment": "Is this still needed or can we add the citation object here?"
+    },
+    "dwc:identificationVerificationStatus": {
+      "type": "boolean",
+      "description": "If this is the accepted identification, based on https://rs.tdwg.org/dwc/terms/identificationVerificationStatus",
+      "example": [
+        true
+      ]
+    },
+    "dwc:identificationRemarks": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identificationRemarks",
+      "examples": [
+        "Distinguished between Anthus correndera and Anthus hellmayri based on the comparative lengths of the uñas."
+      ]
+    },
+    "dwc:identificationQualifier": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identificationQualifier",
+      "examples": [
+        "cf. var. oxyadenia"
+      ]
+    },
+    "???:typeDesignationType": {
+      "type": "string",
+      "description": "Unclear yet",
+      "$comment": "Unknown what this field should be"
+    },
+    "???:typeDesignatedBy": {
```

Comment:
it was proposed here but no other references: https://github.com/dshorthouse/agents_actions/issues/2

Reply:
We need to rework agents to a generic object, then this field will probably become obsolete